tutormfe_extensions/plugin.py

```diff
@@ -42,7 +67,78 @@
         # Danger zone!
         # Add values to override settings from Tutor core or other plugins here.
         # Each override is a pair: (setting_name, new_value). For example:
-        ### ("PLATFORM_NAME", "My platform"),
+        (
```

Comment:
I think we can use the `CORE_MFES_CONFIG` that I said before in here like this:

```python
[ (k,v) for k,v in CORE_MFES_CONFIG.items() ]
```

Where `CORE_MFES_CONFIG` is a dict with the setting like this:

```python
CORE_MFES_CONFIG = {
    "MFE_LEARNING_MFE_APP": {
        "name": "learning",
        "repository": "https://github.com/eduNEXT/frontend-app-learning",
        "port": 2000,
        "version": "ednx-release/palma.master",
    },
}
```